src/components/common/Navbars/NavBar.js

```diff
@@ -26,6 +25,8 @@ import NavBarLinks from './NavBarLinks';
 
 import { connect } from 'react-redux';
 
+import navLogo from '../../../img/navbar-logo.png';
+
```

Comment:
I love that the logo was added, but I am seeing a weird bunch of space between the logo and the nav links. That may be just me though.

Reply:
I was contemplating the space between it too, especially because the text has been removed. I definitely want to start a PR for  continue working on this and also fixing some responsive that broke as well. Would love to hear your suggestion with the nav bar links and logo spacing.